capable/ffi/types/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2022 MobileCoin Inc.
+
+//! FFI type bindings for the types used by libsgx_capable.{so,a}.
+
+#![no_std]
```

Comment:
Similar to the sys crate, not sure no_std is needed
```suggestion
```

Reply:
Alternately, if we don't need `std`, why not be `no_std`? What does it cost us?

Reply:
`foo-types` crates in this repo should always be `no_std` (I left it here), so we can use those structures inside the enclave. Actual FFIs may or may not, depending on whether the libs they're linking to are "capable" of it... 🥁